files/docker/db-sync/guildsetup/scripts/docker-entrypoint.sh

```diff
@@ -1,9 +1,19 @@
 #!/usr/bin/env bash
 
-apk update 2>/dev/null
-apk add socat curl gawk jq 2>/dev/null
+###################### Customisations - START ##################################
+apt-get update 2>/dev/null
+apt-get install -y socat curl gawk jq sudo 2>/dev/null
+
+# Sets the postgres user password unlocking sudo privileges
+# Should be improved to use env variables or similar
+echo -e "postgres\npostgres" |  passwd postgres
```

Comment:
Default postgres setup deploy allows setting up db in a way that creds are not really needed (which is how guide instructions atm are), preferring socket

Reply:
Hm, yes, I suppose we can duplicate that on docker as well... This was just a quick fix to get it going.